stock_picking_move_line_descriptions/models/stock_move_line.py

```diff
@@ -0,0 +1,10 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+    
+    description = fields.Char('Description', index=True, related='move_id.name', required=True)
```

Comment:
Suggested:
```suggestion
    description = fields.Char(related='move_id.name')
```
- `string` attribute is automatically resolved by Odoo
- `index` is not required here IMO
- since this field is related read only, required has no sense.